tests/src/Traits/UtilityTrait.php

```diff
@@ -72,6 +72,40 @@ protected function assertNotVisuallyVisible(NodeElement $element): void {
     Assert::assertFalse($this->isVisuallyVisible($element), 'The element is not visually visible');
   }
 
+  /**
+   * Creates an XPath selector to match by class.
+   *
+   * @param string $class
+   *   The class.
+   * @param bool $wrap
+   *   When true, wraps the expression between square brackets to be used
+   *   directly as selector. Defaults to true.
+   *
+   * @return string
+   *   The xpath selector.
+   */
+  protected function xpathHasClassSelector(string $class, bool $wrap = TRUE): string {
```

Comment:
This function name makes it seem like an assertion, you would expect it to return a boolean `TRUE` if the selector is present.

BTW I would not use functions to compose XPath selectors, this just makes it more opaque and confusing. Mink is also doing this and it is so difficult to read because you never see the full XPath and instead need to jump between functions to mentally assemble it.

Reply:
Done.